### ISSUE TITLE
Widgets should allow enabled, visible at initialisation

### ISSUE THREAD

**Issue by martinohanlon** [lawsie/guizero#90]
Widgets which can be enabled, disabled, show, hidden should also allow these values to be set at initilisation, e.g. 

Instead of:

```python
button_on = PushButton(app, on, text="On")
button_on.disable()
```

use

```python
button_on = PushButton(app, on, text="On", enabled = False)
```

**Comment by lawsie** [comment]
Mmm, I'm sort of inclined to agree that this might be too much. The reason I didn't bother implementing hide/show etc. at all to begin with is that I wanted to keep things as simple as possible. It's nice to have lots of stuff you can do but for the vast majority of kids and beginner users they won't need to use any of these methods so it might be best to just leave them as methods for more advanced users. 

I guess my philosophy with guizero is to keep things as simple as possible and not overwhelm people with too much choice!

**Comment by martinohanlon** [comment]
Enabled and visible 'feel' like states which should be set at initialisation and enabling the state of components feels like a useful and widely used use-case.

A simple app with a Start/Stop (or Connect/Disconnect or Open/Close or Horizontal/Vertical) would want to start with 1 button enabled and 1 disabled.

By using defaults the api can be extended offering additional functionality without adding complexity, creating an enabled button is no more complicated than it ever was, whereas you have the option of creating a disabled button at initialisation.

```python
enabled_button = PushButton(app, command)
disabled_button = PushButton(app, command, enabled = False)
```

Its clearly not a priority, but its a simple change and adds some functionality.

**Comment by Coal0** [comment]
Welp, you've convinced me.

**Comment by m4ddav3** [comment]
Hi,
I recently came across this framework after reading about it in (Hello World) magazine at a school, and immediately used it for a Langton's Ant exercise and to get myself back into Python.

I've started implementing the enabled=boolean functionality on the widgets as I needed it to make my interface feel move intuitive. I later came across this very issue describing it. I still have to complete it for the ButtonGroup and MenuBar widgets.

You have mentioned visibility as well but it is unclear if that functionality is required..?
I agree that visibility feels like a state that should be controllable, but I'm unsure of the ultimate goal here.
